test/unittests/mongoBackend/mongoUpdateContextGeo_test.cpp

```diff
@@ -207,6 +207,7 @@ TEST(mongoUpdateContextGeoRequest, newEntityLocAttribute)
     EXPECT_FALSE(a2.hasField("creDate"));
     EXPECT_FALSE(a2.hasField("modDate"));
     EXPECT_STREQ("A1", ent.getObjectField("location").getStringField("attrName"));
+    ASSERT_TRUE(ent.hasField("location"));
```

Comment:
Why a segfault?
Pretty much the reason for having asserts is to **avoid** segfaults, and get a cemprehensive error message instead ...
Is it not possible (in a not too difficult way) to avoid having a segfault?